Zend/zend_generators.stub.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+final class Generator implements Iterator
+{
+    /** @return void */
+    function rewind() {}
+
+    /** @return bool */
+    function valid() {}
+
+    function current() {}
+
+    /** @return int|string */
+    function key() {}
+
+    /** @return void */
+    function next() {}
+
+    function send($value) {}
```

Comment:
Wouldn't `mixed` type hints provide any added value? I hope that it gets added to PHP in the not too distant future :)

Reply:
I'm not convinced that adding *explicit* `mixed` type annotations would be beneficial.